src/scaffoldfitter/fitter.py

```diff
@@ -421,9 +439,51 @@ def _defineCommonDataFields(self):
                 self._activeDataProjectionGroupFields.append(activeDataProjectionGroupField)
                 self._activeDataProjectionMeshGroups.append(activeDataProjectionGroupField.getOrCreateMeshGroup(mesh))
 
+    def transferData(self, dataRegion):
```

Comment:
Is this functionality availble in the cmlibs.utils package? If it isn't it should be?

Reply:
Moved to cmlibs.utils.